src/main.go

```diff
@@ -6,9 +6,10 @@ import (
 	"Scruticode/src/functions/arguments"
 )
 
+// https://stackoverflow.com/questions/56039154/is-it-really-bad-to-use-init-functions-in-go#56039373
+// https://leighmcculloch.com/posts/tool-go-check-no-globals-no-inits/
```

Comment:
```suggestion
// https://stackoverflow.com/questions/56039154/is-it-really-bad-to-use-init-functions-in-go#56039373
// https://leighmcculloch.com/posts/tool-go-check-no-globals-no-inits/
```
Estos enlaces deberían estar en la descripción del template del PR con una breve explicación o simplemente referenciados, pero tenerlos aquí no aportará a futuro, habrán 2 links asociados a buenas o malas prácticas de Go, pero no tendrás `init functions` en el archivo main...

Reply:
Pensaba que si lo dejo en el PR se van a perder mientras que si quedan en el main me queda de recordatorio de no usar la funcion init que crea side effects con variables globales (cosa que habia hecho pero un linter se quejo)